src/components/map/MapComponent.tsx

```diff
@@ -32,7 +32,7 @@ export class MapComponent extends Component<PropsFromStore & MapApi & WithTransl
         width: 0,
         height: 0,
         visibleLinks: defaultVisibleRelationsLinks,
-        legendIsVisible: true,
+        legendIsVisible: false,
```

Comment:
I like this idea, can you please persist this flag into local storage? example [code here ](https://github.com/nurikk/zigbee2mqtt-frontend/blob/f0a2cf573f5a2d09d8262de2ba958fd534dba55e/src/components/grid/persist.tsx#L9)